projects/admin/src/app/app.component.html

```diff
@@ -15,12 +15,14 @@
   along with this program.  If not, see <http://www.gnu.org/licenses/>.
 -->
 <ngx-spinner type="ball-spin-clockwise" size="medium"></ngx-spinner>
-<div *ngIf="!user" translate>Loading…</div>
-<ng-container *ngIf="user">
+<ng-template #usernotloaded>
+  <div translate>Loading…</div>
+</ng-template>
```

Comment:
Not sure, but I think ngx-spinner can handle a message like this.

Reply:
done